src/mumble/MainWindow.cpp

```diff
@@ -506,28 +506,52 @@ bool MainWindow::nativeEvent(const QByteArray &, void *message, long *) {
 void MainWindow::closeEvent(QCloseEvent *e) {
 #ifndef Q_OS_MAC
 	ServerHandlerPtr sh = Global::get().sh;
-	if (sh && sh->isRunning() && Global::get().s.bAskOnQuit && !bSuppressAskOnQuit) {
+	if (sh && sh->isRunning() && !bSuppressAskOnQuit
+		&& Global::get().s.wcqbWhenConnectedQuitBehaviour == Settings::WhenConnectedQuitBehaviourAsk) {
 		QMessageBox mb(QMessageBox::Warning, QLatin1String("Mumble"),
 					   tr("Mumble is currently connected to a server. Do you want to Close or Minimize it?"),
 					   QMessageBox::NoButton, this);
+		QCheckBox   *qcbRemember = new QCheckBox("Remember this setting");
```

Comment:
```suggestion
		QCheckBox   *qcbRemember = new QCheckBox(tr("Remember this setting"));
```
This makes the string translatable